packages/tools/pie-wrapper-react/react-remove-wrapper-config.js

```diff
@@ -0,0 +1,16 @@
+import removeReactWrapper from './scripts/react-remove-wrapper.js';
+import fs from 'fs-extra';
+import path from 'path';
+
+const componentPath = path.resolve(process.cwd(), './custom-elements.json')
+
+// fetches custom-elements.json file
+const loadJSON = (path) => JSON.parse(fs.readFileSync(path));
+
+export default {
+    plugins: [
+        removeReactWrapper(
```

Comment:
Since these files can actually perform changes, I wouldn't name them `*-config`. Usually config related files only provide settings, not much else.

Reply:
ah thanks! I've actually been able to refactor the code to remove this and the `add-wrapper-config` entirely 👍 the package.json script now heads directly to the react-wrapper scripts themselves. I originally got the config approach from the adobe example but I don't think it's needed in our case 😄